docs/fundamentals/apicompat/package-validation/baseline-version-validator.md

```diff
@@ -120,7 +120,7 @@ In this example, the *CompatibilitySuppressions.xml* contains the suppression fo
 
 This file should be checked into source control to document and review the breaking changes made in a PR and the upcoming release.
 
-After you have released version 2.0.0 of the package, you would delete the *CompatibilitySuppressions.xml* file and update the `PackageValidationBaselineVersion` property to validate future changes against the new release.
+After you've released version 2.0.0 of the package, you can delete the *CompatibilitySuppressions.xml* file and update the `PackageValidationBaselineVersion` property to validate future changes against the new release.
```

Comment:
Commenting on a random line. Baseline package validation now also has a strict mode which we should probably document (similar to the other package validator documents): `EnableStrictModeForBaselineValidation`.

You want to set this to true when you want the apicompat to perform "equality" checks instead of compatibility checks. This means that api additions or changing the assembly (in a compatible way) is disallowed. The primary use-case for this is servicing in which API additions are usually forbidden.

You could also enable this so that APICompat recores all compatibility differences in the suppression file (for tracking API changes). For that you would also need to set `ApiCompatGenerateSuppressionFile` to true.

Reply:
I'll track that separately in https://github.com/dotnet/docs/issues/38551.